src/Functions/initcap.cpp

```diff
@@ -0,0 +1,66 @@
+#include <Functions/FunctionFactory.h>
```

Comment:
Basically I copied from [LowerUpperImpl.h](https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/src/Functions/LowerUpperImpl.h) and [LowerUpperUTF8Impl.h](https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/src/Functions/LowerUpperUTF8Impl.h)